test/index.test.ts

```diff
@@ -1,31 +1,58 @@
 import { strictEqual, deepStrictEqual } from "assert";
 import { describe, test } from "mocha";
-import { getAccounts, getConnection } from "@tableland/local";
+import { getAccounts, getDatabase, getValidator } from "@tableland/local";
 
 describe("index", function () {
   // Note that we're using the second account here
   const [, signer] = getAccounts();
-  const sdk = getConnection(signer);
+  const db = getDatabase(signer);
+  const validator = getValidator();
 
   test("create", async function () {
-    const { name } = await sdk.create("counter integer", { prefix: "table" });
-    strictEqual(name, "table_31337_2");
+    const { meta } = await db
+      .prepare("CREATE TABLE my_table (counter integer);")
+      .run();
+    strictEqual(meta.txn?.name, "my_table_31337_2");
   });
 
   test("insert", async function () {
-    const { hash } = await sdk.write("insert into table_31337_2 values (1);");
-    const txnReceipt = await sdk.receipt(hash);
+    const { meta } = await db
+      .prepare("insert into my_table_31337_2 values (1);")
+      .run();
+
+    if (typeof meta.txn?.transactionHash !== "string") {
+      throw new Error("something unexpected happened during insert");
```

Comment:
This will never be thrown, but we have to put a check here since typescript will not build because of type mismatch

Reply:
Correct. But it isn't throwing because of a type mismatch per se. It is just trying to protect us from doing something with something that might be undefined. So we just have to use type guards like this to be safe.